### ISSUE TITLE
[11/2000] Equip slot is already in use

### ISSUE THREAD

**Issue by Matt-Swift** [fuzziqersoftware/newserv#301]
A player attempted to play 11/2000 and ran into the below issue... if you need more of a log, I can provide it.

```
I 2019812 2023-12-28 16:59:05 - [Commands] Received from C-3F (mistamontiel) (version=DC_V1_11_2000_PROTOTYPE command=60 flag=00)                                                                                                           
0000 | 60 00 10 00 27 03 00 00 0D 00 01 00 01 00 00 00 | `   '                                                                                                                                                                              
I 2019812 2023-12-28 16:59:05 - [Lobby:1B] Player 0 destroyed inventory item 0:0001000D (Telepipe x1)                                                                                                                                       
[PlayerInventory] Meseta: 290419                                                                                                                                                                                                            
[PlayerInventory] 19 items
[PlayerInventory]    0: [+00000010] 03010000 00070000 00000000 (00010000) 00000000 (Monofluid x7)
[PlayerInventory]    1: [+0000000C] 01020000 00000000 00000000 (00010001) 5B006280 (Barrier)
[PlayerInventory]    2: [+00000004] 00010000 00000214 00000000 (00010002) 31004680 (Saber 0/20/0/0/0)
[PlayerInventory]    3: [+0000000C] 021B2814 82034A08 D903D100 (00010003) 6FC20306 (Sumba LV40 8.98/21.22/9.85/2.09 111% 194IQ PB:L,G (white))
[PlayerInventory]    4: [+0000000C] 01010000 00000000 00000000 (00010004) 5B006280 (Frame)
[PlayerInventory]    5: [+00000004] 00010001 00000114 00000000 (00010005) 00000000 (Saber +1 20/0/0/0/0)
[PlayerInventory]    6: [+00000004] 00060001 0000020F 00000000 (00010007) 00000000 (Handgun +1 0/15/0/0/0)
[PlayerInventory]    7: [+00000004] 01010300 00000000 00000000 (00010008) 00000000 (Giga Frame)
[PlayerInventory]    8: [+00000000] 03070000 00090000 00000000 (0001000D) 00000000 (Telepipe x9)
[PlayerInventory]    9: [+00000000] 03010100 00020000 00000000 (00001037) 00000000 (Difluid x2)
[PlayerInventory]   10: [+00000000] 00010104 1B000000 00000000 (0001000F) 60A72841 (Shock Brand +4)
[PlayerInventory]   11: [+00000000] 03060100 00010000 00000000 (00000FF1) 00000000 (Antiparalysis x1)
[PlayerInventory]   12: [+00000000] 03000000 00010000 00000000 (00000FEF) 00000000 (Monomate x1)
[PlayerInventory]   13: [+00000000] 00060100 00000000 00000000 (00000FF0) 00000000 (Autogun)
[PlayerInventory]   14: [+00000000] 00030500 80000000 00000000 (00000458) 00000000 (???? BLADE DANCE)
[PlayerInventory]   15: [+00000000] 00030500 80000000 00000000 (00000459) 00000000 (???? BLADE DANCE)
[PlayerInventory]   16: [+00000000] 00010103 00000314 00000000 (00001060) 00000000 (Brand +3 0/0/20/0/0)
[PlayerInventory]   17: [+00000000] 01010300 00000000 00000000 (0000106C) 00000000 (Giga Frame)
[PlayerInventory]   18: [+00000000] 00080000 1B00021E 00050005 (00010006) 00000000 (Shock Mechgun 0/30/0/0/0)
I 2019812 2023-12-28 16:59:05 - [Commands] Received from C-3F (mistamontiel) (version=DC_V1_11_2000_PROTOTYPE command=60 flag=00)
0000 | 60 00 20 00 60 07 FB 8C 00 00 06 00 79 00 00 00 | `   `       y   
0010 | 55 9D D2 44 00 00 C0 40 A7 DE 95 44 BD 47 00 00 | U  D   @   D G  
I 2019812 2023-12-28 16:59:12 - [Commands] Received from C-3F (mistamontiel) (version=DC_V1_11_2000_PROTOTYPE command=60 flag=00)
0000 | 60 00 10 00 23 03 00 00 06 00 01 00 00 00 00 00 | `   #           
I 2019812 2023-12-28 16:59:12 - [C-3F] Equipped item 00010006
I 2019812 2023-12-28 16:59:14 - [Commands] Received from C-3F (mistamontiel) (version=DC_V1_11_2000_PROTOTYPE command=60 flag=00)
0000 | 60 00 10 00 23 03 00 00 F0 0F 00 00 00 00 00 00 | `   #           
W 2019812 2023-12-28 16:59:14 - [Server] Error processing client command: equip slot is already in use
I 2019812 2023-12-28 16:59:14 - [Server] Client disconnected: C-3F on fd 39
```

**Comment by fuzziqersoftware** [comment]
Yeah, I'd like to see a full game log if possible - the inventory item IDs are very unusual here.

**Comment by Matt-Swift** [comment]
Here it is: 
[proto.txt](https://github.com/fuzziqersoftware/newserv/files/13788234/proto.txt)

**Comment by fuzziqersoftware** [comment]
Thanks - apparently the odd item IDs are correct, and it's just a thing that DC NTE and 11/2000 do.

The equip issue has been fixed by 29a4347f. The problem was that all later versions send an unequip command before the equip command if you switch equipped items, but NTE and 11/2000 don't. The server now handles this properly.
